src/Microsoft.ML.Ensemble/Selector/SubsetSelector/BaseSubsetSelector.cs

```diff
@@ -20,23 +20,23 @@ public abstract class ArgumentsBase
         }
 
         protected readonly IHost Host;
-        protected readonly TArgs Args;
+        protected readonly TOptions BaseSubsetSelectorOptions;
```

Comment:
>BaseSubsetSelectorOptions [](start = 36, length = 25)

Why not `Options`?

Reply:
(plz see above)

---
In reply to: [258280102](https://github.com/dotnet/machinelearning/pull/2636#discussion_r258280102) [](ancestors = 258280102)